src/contracts/deposits-registry/sanity-checker/integrity-checker/integrity-checker.service.ts

```diff
@@ -116,7 +111,7 @@ export class DepositIntegrityCheckerService {
     eventsCache: VerifiedDepositEvent[],
   ) {
     for (const [index, event] of eventsCache.entries()) {
-      tree.insertNode(event.depositDataRoot);
+      tree.insert(event.depositDataRoot);
 
       if (index % DEPOSIT_TREE_STEP_SYNC === 0) {
         await new Promise((res) => setTimeout(res, 1));
```

Comment:
below we have log:

 this.logger.log('Checking integrity of saved deposit events', {
          processed: index,
          remaining: eventsCache.length - index,
        });
   But we don't really Checking integrity 
   instead we Inserting deposit events
   
   will suggest `Inserting verified deposit events`